LEGO1/mxdirectx/mxdirect3d.cpp

```diff
@@ -614,10 +614,205 @@ MxDeviceEnumerate::DirectDrawEnumerateCallback(LPGUID p_guid, LPSTR p_driverDesc
 // STUB: LEGO1 0x1009c730
```

Comment:
This should no longer be a STUB?

Reply:
No indeed

Reply:
I'll fix it when I reach a PC (unless you're first)